Hazel/src/Platform/Windows/WindowsWindow.cpp

```diff
@@ -41,7 +41,6 @@ namespace Hazel {
 
 		if (s_GLFWWindowCount == 0)
 		{
-			HZ_CORE_INFO("Initializing GLFW");
```

Comment:
We could keep this in. Otherwise we should probably also remove the extra logging in `EntryPoint` (the "Hello! Var = ..." thing).

Reply:
Cherno actually specifically said he wanted these logs gone, they aren't randomly being removed.

Reply:
![image](https://user-images.githubusercontent.com/26593477/68541665-b2f1d580-03a2-11ea-9b28-0a256c6abc7f.png)

The reasoning for this: this is pure something the engine needs to do, the user doesn't have any knowledge of GLFW or it's existence. So having a message telling it is initializing is just cluttering the logging. This was good for testing the PR about the shutdown and the windowcount. But not something to leave in the code.

However, we do have an assert to notify when it goes wrong, to give feedback to Hazel about why the application is not launching.

I'll mark this as resolved.